### PR TITLE
Dark mode 🕶️

### LINKED ISSUE #185: Dark mode
Use [CSS variables](https://www.youtube.com/watch?v=rXuHGLzSmSE) to toggle between light and dark mode. Note: integration with Bulma could be a bit tricky.

### PR DESCRIPTION
**Pull request by triszt4n**
Closes #185 

### What's new:
* Fancy, cool, lovable stylesheet for the comfort of our weary eyes. 💖
* Uses darkly scss from bulmaswatch, but redesigned to resemble original bulma (like light theme).
* Fancy button up in the navbar ✨ 
* Overrides style of bulma-calendar, easymde and fullcalendar (so proud of this).
* localStorage stores the settings (lovely invention, more [here](https://www.w3schools.com/html/html5_webstorage.asp))
* Only easymde's textarea is left untouched (codeMirror has it's own styles)

## Enjoy! 🥳

### CODE REVIEW

**Comment by OmTheTurtle** [comment]
Very nice work, looks great! One small thing, is it possible to extract the common rules from `main-dark.scss` and `main-light.scss` (I saw you already did it with some in `additions.scss`)? There are some repeated things like the `html`, `body` and the alert animation.

File: public/js/themes.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+// Managing theme changes:
+const darkId = 'dark-theme-css'
+const lightId = 'light-theme-css'
+
+function setDarkCss() {
```
Comment (poiuztrewq1) [inline public/js/themes.js:5]: I really like the dark theme.
Maybe `setDarkCss` and `setLightCss` could be done in one function with a paramteter as they are doing basicly the same thing.

Comment (triszt4n) [reply public/js/themes.js:5]: Yep, totally true, forgot to do that, hotfix on it's way.

File: public/js/themes.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+function changeCssToDark(toDark) {
+  if (toDark) {
+    darkEl.media = ''
+    lightEl.media = 'none'
+  }
+  else {
+    lightEl.media = ''
+    darkEl.media = 'none'
+  }
+}
```
Comment (falusi94) [inline public/js/themes.js:16]: I like the idea of DRY-ing up the two functions by moving the two selectors in the upper scope, but I'd avoid the flag argument in this case, here is a good explanation on that.

https://martinfowler.com/bliki/FlagArgument.html

**Pull request merged**